<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-02T17:20:58 IST" name="Suite" finished-at="2024-05-02T17:21:29 IST" duration-ms="31025">
    <groups>
    </groups>
    <test started-at="2024-05-02T17:20:58 IST" name="Test" finished-at="2024-05-02T17:21:29 IST" duration-ms="31025">
      <class name="com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test@79924b]" started-at="2024-05-02T17:20:59 IST" name="setup" finished-at="2024-05-02T17:21:05 IST" duration-ms="5944" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="jobapplication()[pri:0, instance:com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test@79924b]" started-at="2024-05-02T17:21:05 IST" name="jobapplication" finished-at="2024-05-02T17:21:29 IST" duration-ms="24423" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [37300] but found [2921.25]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [37300] but found [2921.25]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.AcharyaUniversity_ERP.Pageobject.JobApplicationPage_University.comnmethodtovalidatesalarybreakup(JobApplicationPage_University.java:740)
at com.AcharyaUniversity_ERP.Pageobject.JobApplicationPage_University.salaryBreakUp_validation(JobApplicationPage_University.java:510)
at com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test.jobapplication(JobApplication_Univesity_Test.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jobapplication -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test@79924b]" started-at="2024-05-02T17:21:29 IST" name="teardown" finished-at="2024-05-02T17:21:29 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.AcharyaUniversity_ERP.Testcase.JobApplication_Univesity_Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
